themes/default/content/blog/aws-static-website-component/index.md

```diff
@@ -0,0 +1,188 @@
+
+## What the Component Does
+
+As we said, the component deploys a static website to AWS and, optionally, a CloudFront CDN. You simply add a few lines of code, set a few configuration parameters, run `pulumi up` and let the component do the rest. The component sets up:
+
+* An Amazon S3 bucket
+* An Amazon CloudFront CDN (optional)
+    * An Amazon Route 53 DNS record
+    * An Amazon Certificate Manager
+
+## Some Examples
+
+Let’s look at two examples of how to use the component. You’ll find both examples and more at [https://gist.github.com/sean1588/f6ca2e09550e53c869259d569f29e9a1](https://gist.github.com/sean1588/f6ca2e09550e53c869259d569f29e9a1). We’re going to use YAML in this post but the gist also contains the JSON equivalents.
+
+## Create a Website
+
+This YAML code creates a basic static website.
+
+```yaml
+name: react-static-website
+runtime: yaml
+description: Example of deploying React with Pulumi
+resources:
+  web:
+    type: "aws-static-website:index:Website"
+    properties:
+      sitePath: "../build"
+outputs:
+  websiteURL: ${web.websiteURL}
+```
+
+This example code deploys a static React website but you can use the component with any other static website framework, such as Hugo. The only parameter that would change is the output directory specified in the** sitePath** property.
+
+The first two lines of this example give the project a name and tell Pulumi that the code is YAML. The next line is simply a description of the project.
+
+Next you allocate a _resource_. This is a Pulumi term. Resources are the fundamental units that make up your cloud infrastructure and Pulumi provides many resources that are ready for you to use. Here, you only need one: **aws-static-website:Website**, which is the AWS Static Website component. The resource has an input property, **sitePath**, which is the root directory that contains the website's contents. Of course, you can use any path you want.
+
+Finally, you’ll probably want to know the URL of your new site once it’s deployed. That’s what the **outputs** line does. If you’ll remember, **websiteURL** is one of the outputs that the component provides.
+
+Remember, even though there are a few terms in the code you might be unfamiliar with, you can simply cut and paste the code in the gist.
+
+### Get Started
+
+To get started, go to the folder that contains your static website files. Then, follow these steps.
+
+1. Create a directory named **infrastructure** with this command:
+
+```bash
+mkdir infrastructure && cd infrastructure
+
+```
+
+1. You’re now in the **infrastructure** directory. Generate a **Pulumi.yaml** file with this command:
+
+```bash
+pulumi new aws-yaml
+```
+
+Here’s an example of what the final directory structure looks like.
+
+```
+| react-website
+├── build
+│   ├── index.html
+│   ├── main.css
+├── infrastructure
+│   ├── Pulumi.yaml
+```
+
+**Note:** Pulumi will add some other files to the **infrastructure** directory but you can ignore them.
```

Comment:
I definitely agree and did mention this to Roberta during the review and she felt it would be best to word it this way as to not distract the user and not need to go into an explanation of stacks. I'll see if I can come up with a rewording of this, because I also feel we shouldn't be saying to just ignore the files.